public/pages/Findings/components/FindingDetailsFlyout.tsx

```diff
@@ -83,16 +83,16 @@ export default class FindingDetailsFlyout extends Component<
     };
   }
 
-  componentDidMount(): void {
-    this.getIndexPatternId().then((patternId) => {
-      if (patternId) {
-        this.setState({ indexPatternId: patternId });
-      }
-    });
-
+  getCorrelations = async () => {
     const { id, detector } = this.props.finding;
-    const allFindings = this.props.findings;
-    DataStore.correlationsStore
+    let allFindings = this.props.findings;
+    if (allFindings.length === 1 && allFindings[0].id === id) {
```

Comment:
We shouldn't take the count as the source of truth here since this can change anytime and also this will be true if there is only finding in the system.

Let's use a prop to check the source of the open action

Reply:
Gotcha, let me refactor this part.

Reply:
Done